src/components/MenuButton/index.styled.ts

```diff
@@ -0,0 +1,29 @@
+import styled, { css } from 'styled-components';
+
+const MenuButton = styled.div`
+  display: inline-flex;
+  flex-direction: column;
+  gap: 10px;
+  text-align: center;
+
+  width: 100px;
+`;
+
+const Label = styled.label`
+  color: ${({ theme: { colors } }) => colors.gray001};
+
+  font-size: 15px;
+`;
+
+const Button = styled.button`
+  width: 100%;
+  height: 20px;
+  border-radius: 50px;
+
+  ${({ canBuy }: { canBuy: boolean }) => css`
+    background: ${({ theme: { colors } }) =>
+      canBuy ? colors.brown001 : colors.gray002};
+  `}
```

Comment:
button html element 를 사용하고 있으니, 우리 html element 가 주는 네이티브 속성을 최대한 활용해보아요!

아예 disabled attribute 를 넘겨주고  , disabled 속성 스타일링을 정의해주는건 어떨까요..?!
```suggestion
const Button = styled.button`
  width: 100%;
  height: 20px;
  border-radius: 50px;
   background: ${({ theme: { colors } }) =>
      colors. brown001};

 &: disabled { 
        background: ${({ theme: { colors } }) =>
      colors.gray002};
 }
```

사용되는 곳

```tsx

<Button disabled={ //... true or false }/>
```

Reply:
오 너무 좋은데요?!?! 왜 하면서는 그 생각을 못했을까여,, 🥲
바로 수정하겠습니당 친절한 예시 코드까지,, 캄사합니둥 🥰 [commit](https://github.com/woowacourse-study/2022-looking-for-the-sound-of-an-object/pull/18/commits/8c2a1be1173225d62443316721c0b6615b4b5e1f)